components/x-gift-article/src/SharedLinkTypeSelector.jsx

```diff
@@ -70,14 +75,29 @@ export const SharedLinkTypeSelector = (props) => {
 						href={`${enterpriseEnabled ? 'mailto:customer.success@ft.com' : 'mailto:help@ft.com'}`}
 						rel="noreferrer"
 						target="_blank"
-						data-trackable="enterprise-out-of-credits"
-					>
+						data-trackable="enterprise-out-of-credits">
 						contact support
 					</a>
 					.
 				</NoCreditAlert>
 			)}
 			{showAdvancedSharingOptions && <AdvancedSharingOptions {...props} />}
+			{hasHighlights && (
```

Comment:
**Question:** this should be limited just to B2B users - only B2B users should see the option to share highlights... is that the case? I know only B2B users can create highlights at the moment, but if a B2C user receives an article that contains highlights and then re-shares it, what would happen? Would they get the option to share the highlights? Would that cause an error if they did?

Reply:
yeah, you are right, this would allow B2C customers to share highlights if it was shared to them , and it may cause issues since the owner of the highlights share  token and the article shared would be different in this case which will cause issues with redemption count for both of them